### PR TITLE
review code snippet

### PR DESCRIPTION
**Pull request by Gustry**
### Description

Goal: review code snippet

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/pyqgis_developer_cookbook/plugins/snippets.rst [hunk 1]
```diff
@@ -22,39 +20,38 @@ In the plug-in add to the :func:`initGui()`
...
 
 .. index:: Plugins; Toggle layers
 
 How to toggle Layers
 --------------------
 
-Since QGIS 2.4 there is new layer tree API that allows direct access to the
-layer tree in the legend. Here is an example how to toggle visibility of the
-active layer
+There is an API to access to layers in the legend.
```
Comment (havatv) [inline source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: `There is an API to access to layers in the legend.`
 ->
`There is an API to access layers in the legend.`

File: source/docs/pyqgis_developer_cookbook/plugins/snippets.rst [hunk 2]
```diff
@@ -22,39 +20,38 @@ In the plug-in add to the :func:`initGui()`
...
+Here is an example how to toggle visibility of the active layer
```
Comment (havatv) [inline source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: `Here is an example how to toggle visibility of the active layer`
->
`Here is an example that toggles the visibility of the active layer`

File: source/docs/pyqgis_developer_cookbook/plugins/snippets.rst [hunk 3]
```diff
@@ -63,33 +60,33 @@ How to access attribute table of selected features
...
-
-
-The method requires one parameter (the new value for the attribute
+   def change_value(value):
+       """Change the value in the second column for all selected features.
+
+       :param value: The new value.
+       """
+       layer = iface.activeLayer()
+       if(layer):
```
Comment (havatv) [inline source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: `if(layer):`
->
`if (layer):`

File: source/docs/pyqgis_developer_cookbook/plugins/snippets.rst (hunk 1 above)
Comment (Gustry) [reply source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: done

File: source/docs/pyqgis_developer_cookbook/plugins/snippets.rst (hunk 2 above)
Comment (Gustry) [reply source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: done

File: source/docs/pyqgis_developer_cookbook/plugins/snippets.rst (hunk 3 above)
Comment (Gustry) [reply source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: parenthesis are redundant. I removed them

Comment (havatv) [reply source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: My point was to use the same "style" through the example.

Comment (Gustry) [reply source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: Sorry, can you develop?
Are you talking about this example specifically or about all examples in the cookbook?

It's already the same *style* here, no?

Comment (havatv) [reply source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: It is fine now. I just wanted to say what my comment was about (I did not want to express any style preference).
We can of course discuss if there should be a style guideline for the code in the cookbook (but I think we should use our limited resources on other matters now).

Comment (Gustry) [reply source/docs/pyqgis_developer_cookbook/plugins/snippets.rst]: No problem ;-)
I was updating these snippets, but at the same time, I'm also following Python PEP 8 and PEP 257.

PEP8 is for code formatting: https://www.python.org/dev/peps/pep-0008/
And PEP257 for docstring formatting

**Pull request merged**